person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  att_reader :id
+
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Please put this method under `private` you created below to make it a private method.
It will be like that:
```ruby
private

 def of_age?
    @age >= 18
  end
```